src/goose/toolkit/developer.py

```diff
@@ -39,9 +39,15 @@ def system(self) -> str:
         """Retrieve system configuration details for developer"""
         hints_path = Path(".goosehints")
         system_prompt = Message.load("prompts/developer.jinja").text
+        home_hints_path = Path.home() / ".config/goose/.goosehints"
```

Comment:
It doesn't seem necessary to use a hidden file when it's already in a hidden folder. It might be good to log an error if the dotfile exists though, just to avoid confusion.

Reply:
@vfiano yeah can change it, wouldn't be too confusing as it is called .goosehints elsewhere?